whatsapp.py

```diff
@@ -107,7 +107,7 @@ def count_questions(df, regex=True):
 
 def count_curses(df):
     counter = pd.DataFrame()
-    curse_list = ["פאק", "שיט", "סעמק", "זונה", "דמט", "דאם", "דאמ", "לעזאזל", "זין", "רבאק"]
+    curse_list = ["פאק","פאקינג", "שיט", "סעמק","כוסאמק", "זונה", "דמט", "דאם", "דאמ", "לעזאזל", "זין", "רבאק"]
```

Comment:
We should consider making a txt with curse words by language, instead of a hard coded list. Better practice and also easier to manage